views/pages/ViewChartPage.jsx

```diff
@@ -9,16 +9,34 @@ import UserAvatar from '../components/UserAvatar'
 import ChartFilterForm from '../forms/CharFilterForm'
 import { apiRoutes } from '../routes/routes'
 import api from '../api'
+import UsersModel from '../models/UsersModel'
+import { omitSites } from '../fe-utils/helpers'
 
 const ViewChartPage = () => {
-  const { classes } = useOutletContext()
+  const { classes, user, setNotification } = useOutletContext()
   const { chart_id } = useParams()
   const [graph, setGraph] = useState(null)
+  const allowedSites = omitSites(user.access)
+  const siteOptions = UsersModel.userAccessDropdownOptions(allowedSites)
```

Comment:
can we pass the `user` object here so we don't need the `allowedSites` constant above or the `omitSites` helper here? It feels a bit more natural to pass a user object to a user model function than an array of site strings.

Reply:
this changed [218cfc9](https://github.com/AMP-SCZ/dpdash/pull/441/commits/218cfc99b61b8fbafa89ab15275c64beca1f0b10)